### PR TITLE
Feat/lesq 435/share card container [LESQ-435]

### PR DESCRIPTION
**Pull request by njblore**
## Description

- Create component and story for share card group
- Refactor out some shared logic between download and share cards 
- Rename prop on resource card to be more generic between download/share

## How to test

1. Go to https://deploy-preview-2085--owa-storybook.netlify.thenational.academy/?path=/docs/download-components-share-card-group--docs
2. Click on download components / share card group
3. You should see stories for a few different resources present

## Screenshots

How it should now look:
Screenshot 2023-11-09 at 09 04 51

## Checklist

- [ ] Should match [figma design](https://www.figma.com/file/LSjbzdvHo17XqXGBRNJjQP/Old-%2F-new-content?node-id=4468%3A27528&mode=dev)

### CODE REVIEW

File: src/components/DownloadAndShareComponents/ShareCardGroup/ShareCardGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const sortedResources = sortShareResources(props.shareableResources);
+
+  return (
+    <Flex $flexDirection="column" $gap={[16, 24]} $alignItems="flex-start">
+      <Flex
+        $gap={16}
+        $flexDirection={["column", "row"]}
+        $flexWrap={["nowrap", "wrap"]}
+      >
+        {sortedResources.map((resource) => (
```
Comment (JBR90) [inline src/components/DownloadAndShareComponents/ShareCardGroup/ShareCardGroup.tsx:29]: Should there be a key in this map?

**Review by JBR90 — APPROVED** [review]
Looking good! Approved with a small question

File: src/components/DownloadAndShareComponents/ShareCardGroup/ShareCardGroup.tsx (hunk 1 above)
Comment (njblore) [reply src/components/DownloadAndShareComponents/ShareCardGroup/ShareCardGroup.tsx:29]: yes! good catch

**Review by Monika-Koziol — APPROVED** [review]
Looking good.

**Pull request merged**